compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -920,17 +943,90 @@ abstract class GenJSCode extends plugins.PluginComponent
 
     // Static initializers -----------------------------------------------------
 
-    private def genStaticInitializerLoadingModule(sym: Symbol)(
+    private def genStaticInitializerWithStats(stats: js.Tree)(
         implicit pos: Position): js.MethodDef = {
       js.MethodDef(
           static = true,
           js.Ident(ir.Definitions.StaticInitializerName),
           Nil,
           jstpe.NoType,
-          Some(genLoadModule(sym)))(
+          Some(stats))(
           OptimizerHints.empty, None)
     }
 
+    private def genRegisterReflectiveInstantiation(sym: Symbol)(
+        implicit pos: Position): List[js.Tree] = {
```

Comment:
Historical. At some point this returned several statements. It doesn't anymore, so it should definitely be `Option`.